packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -87,9 +91,26 @@ const ResetPassword = ({
     [navigate]
   );
 
-  const onSubmit = async () => {
-    try {
+  const clearError = useCallback(() => {
+    if (errorText !== '') {
+      setErrorText('');
       setErrorMessage('');
+    }
+  }, [errorText, setErrorText]);
+
+  const localizedEmailRequiredMessage = ftlMsgResolver.getMsg(
+    'reset-password-email-required-error',
+    'Email required'
+  );
+
+  const onSubmit = useCallback(async () => {
+    const email = getValues('email');
+    if (email === '') {
```

Comment:
Q: does this need a `.trim()` to remove potential leading/trailing spaces from `email`, or is it pre-trimmed?